### PR TITLE
USHIFT-1519: hard-code source of LVMS manifests

### PR DESCRIPTION
**Pull request by dhellmann**
The production pipeline is passing an old value. We want that input
parameter to be controlled explicitly by the development, so ignore
the parameter and hard-code a value in the rebase script.

/assign @copejon @ggiguash @pmtk

### CODE REVIEW

File: scripts/auto-rebase/rebase.sh [hunk 1]
```diff
@@ -40,6 +40,8 @@ cluster-csi-snapshot-controller-operator"
 declare -a ARCHS=("amd64" "arm64")
 declare -A GOARCH_TO_UNAME_MAP=( ["amd64"]="x86_64" ["arm64"]="aarch64" )
 
+LVMS_RELEASE_MULTI_ARCH="registry.access.redhat.com/lvms4/lvms-operator-bundle:v4.13.1-5"
```
Comment (ggiguash) [inline scripts/auto-rebase/rebase.sh:43]: Should we allow environment override here?

Comment (dhellmann) [reply scripts/auto-rebase/rebase.sh:43]: The point of this change is to *not* allow anything outside of the script to influence the source of LVMS. I'll provide a better fix as part of USHIFT-1521, but we need something quick so we can get an EC build this sprint.

**Comment by ggiguash** [comment]
/lgtm
/hold

Please unhold the PR when ready to be merged

**Pull request merged**